makefiles/windows.mk

```diff
@@ -0,0 +1,28 @@
+WINDOWS=${OUTPUT}/windows
```

Comment:
This whole file relies on the `dactiv/wix` docker container.  The Wix application generates a lot of intermediate files, so this functions by mounting to the `output/windows` directory and creating the intermediate files until its happy and makes the final `.msi`